ui/webui/src/components/installation/ReviewConfiguration.jsx

```diff
@@ -14,16 +14,125 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with This program; If not, see <http://www.gnu.org/licenses/>.
  */
-import React from "react";
+import cockpit from "cockpit";
+import React, { useEffect, useState } from "react";
 
 import {
-    Form
+    Checkbox, Title,
+    DataList, DataListItem,
+    DataListItemRow, DataListItemCells,
+    DataListCell,
+    DescriptionList, DescriptionListGroup,
+    DescriptionListTerm, DescriptionListDescription,
+    Flex, Popover,
 } from "@patternfly/react-core";
 
+import {
+    getSelectedDisks, getDeviceData,
+} from "../../apis/storage.js";
+
+import {
+    getLanguage, getLanguageData,
+} from "../../apis/localization.js";
+
+import { HelpIcon } from "@patternfly/react-icons";
+
+const _ = cockpit.gettext;
+
 export const ReviewConfiguration = () => {
+    const [deviceData, setDeviceData] = useState({});
+    const [selectedDisks, setSelectedDisks] = useState();
+    const [systemLanguage, setSystemLanguage] = useState("YYY");
+
+    useEffect(() => {
+        getLanguage()
+                .then(res => {
+                    getLanguageData({ lang: res }).then(res => {
+                        setSystemLanguage(res["native-name"].v);
+                    }, console.error);
```

Comment:
That's a good question - I would say it depends on if we have a decent fallback and impact on the installation and target system. Otherwise we might spam the user with cryptic error messages that are hard to act upon.